relay-event-schema/src/processor/attrs.rs

```diff
@@ -123,14 +123,17 @@ impl MaxChars {
             MaxChars::Symbol => 256,
             MaxChars::Path => 256,
             MaxChars::ShortPath => 128,
-            // these are from constants.py or limits imposed by the database
-            MaxChars::Logger => 64,
-            MaxChars::Email => 75,
-            MaxChars::Culprit => 200,
-            MaxChars::TagKey => 32,
-            MaxChars::TagValue => 200,
-            MaxChars::Environment => 64,
-            MaxChars::Distribution => 64,
+            // These limits below were initially derived from Sentry's original implementation in
+            // Python. Before changing any of these limits, check if the values are inserted into
+            // the database by the Sentry processing pipeline, and if there are limits on the
+            // database column.
+            MaxChars::Logger => 64,       // src/sentry/constants.py:???
+            MaxChars::Email => 75,        // src/sentry/constants.py:MAX_EMAIL_FIELD_LENGTH
+            MaxChars::Culprit => 200,     // src/sentry/constants.py:MAX_CULPRIT_LENGTH
+            MaxChars::TagKey => 200,      // src/sentry/constants.py:MAX_TAG_KEY_LENGTH
```

Comment:
Hmmm I'd expect to have a failing test with this change?

Reply:
There was, fixed with [ea62f02](https://github.com/getsentry/relay/pull/2453/commits/ea62f02e96b5557f2fbb827da253af7d08120ede)